src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -160,7 +170,7 @@ class AuthScreens extends React.Component {
             if (isLoadingApp) {
                 App.openApp();
             } else {
-                App.reconnectApp(this.props.lastUpdateIDAppliedToClient);
+                App.reconnectApp(lastUpdateIDAppliedToClient);
```

Comment:
As we are aware already, there's hidden bug where `lastUpdateIDAppliedToClient` is NOT the updated updateID we saved in Onyx as this isn't added in useEffect dependency